chains/evm/contracts/offRamp/OffRamp.sol

```diff
@@ -274,6 +273,17 @@ contract OffRamp is ITypeAndVersion, Ownable2StepMsgSender {
     /////// SECURITY CRITICAL CHECKS ///////
     address receiver = address(bytes20(message.receiver));
 
+    uint256[] memory balancesPreVerification = new uint256[](message.tokenTransfer.length);
```

Comment:
There's only 0 or 1 balances, we can just have a single tokenBalancePre = 0 and populate if there's a token